.env.example

```diff
@@ -1,19 +1,19 @@
+# Required by various APIs
+CLUSTER_ID=fill-this-in
+
 # Required for Administration API tutorial, to acquire auth tokens
 #  See https://docs.camunda.io/docs/apis-tools/administration-api/authentication/#client-credentials-and-scopes
 ADMINISTRATION_CLIENT_ID=fill-this-in
 ADMINISTRATION_CLIENT_SECRET=fill-this-in
-
+ADMINISTRATION_AUDIENCE=api.cloud.camunda.io
 # Required for Administration API tutorial, to access API
 ADMINISTRATION_API_URL=https://api.cloud.camunda.io
-ADMINISTRATION_AUDIENCE=api.cloud.camunda.io
-CLUSTER_ID=fill-this-in
 
 ------------------------------------
 
-# Required for non-Administration API tutorials, to acquire auth tokens
-COMPONENTS_CLIENT_ID=fill-this-in
-COMPONENTS_CLIENT_SECRET=fill-this-in
-
+# Required for Optimize API tutorials, to acquire auth tokens
+OPTIMIZE_CLIENT_ID=fill-this-in
+OPTIMIZE_CLIENT_SECRET=fill-this-in
+OPTIMIZE_AUDIENCE=optimize.camunda.io
```

Comment:
These keys are now specific to the Optimize API *only*. There will be no shared use of these keys across other "component" APIs.

Reply:
I get hung up here if I put on my super novice user hat. If I download my credentials as environment variables, I get values that look like this:

> export ZEEBE_ADDRESS='xx.zeebe.camunda.io:443'
> export ZEEBE_CLIENT_ID='xx'
> export ZEEBE_CLIENT_SECRET=xx'
> export ZEEBE_AUTHORIZATION_SERVER_URL='https://login.cloud.camunda.io/oauth/token'
> export ZEEBE_REST_ADDRESS='https://lhr-1.zeebe.camunda.io/xx'
> export ZEEBE_GRPC_ADDRESS='grpcs://xx-1.zeebe.camunda.io:443'
> export ZEEBE_TOKEN_AUDIENCE='zeebe.camunda.io'
> export CAMUNDA_CLUSTER_ID='xx'
> export CAMUNDA_CLUSTER_REGION='lhr-1'
> export CAMUNDA_CREDENTIALS_SCOPES='Zeebe,Tasklist,Operate,Optimize,Secrets'
> export CAMUNDA_TASKLIST_BASE_URL='https://lhr-1.tasklist.camunda.io/xx'
> export CAMUNDA_OPTIMIZE_BASE_URL='https://lhr-1.optimize.camunda.io/xx'
> export CAMUNDA_OPERATE_BASE_URL='https://lhr-1.operate.camunda.io/xx'
> export CAMUNDA_OAUTH_URL='https://login.cloud.camunda.io/oauth/token'

Zeebe is the only component that shows a `CLIENT_ID` and `CLIENT_SECRET`. This remains true even if the scope is only Optimize.

Do you think this is confusing or misleading?

Reply:
Maybe but I'm not sure how to address the fact that client IDs/secrets _can_ be used across multiple APIs, but they don't have to be, and some of them come from the organization level and others come from the cluster level. Supporting the re-use of credentials across multiple APIs seems to provide multiple points of confusion; assigning one secret/key per API in the environment variables still allows for people to reuse credentials, if they want, but also makes it obvious that they need creds that will work for that specific API.

Do you have any suggestions? Can this be addressed in the supporting tutorial?

Reply:
In the .env.example file, I would note what values you will expect to see in the downloaded credentials. So, in this case, I needed to use the `ZEEBE_CLIENT_ID` as the `OPTIMIZE_CLIENT_ID`, etc. You can also note this in the docs with an admonition. 

I would encourage you or Christina to follow up with the Console team because this looks misleading. I don't think these values are specific to Zeebe but old historical artifacts from a time when there were only Zeebe clients.

Reply:
Is this what you're looking for? 

<img width="1027" alt="image" src="https://github.com/camunda/camunda-api-tutorials/assets/1627089/e22f4c09-b720-4783-b34d-92b3b3317f85">

Reply:
Yes, I think this is super helpful and something we should keep an eye on in case there is confusion around where to get these values. Thank you!

Reply:
@pepopowitz were you going to add these changes to this PR? I'm on the fence about whether this is blocking or not.

Reply:
I thought I had! I just pushed these changes.